### ISSUE TITLE
grunt-sass and phantom versions incompatible with latest node version (4.1.1)

### ISSUE THREAD

**Issue by joelochlann** [guardian/frontend#10775]
When running `npm install` (on its own or as part of `setup.sh`), these dependencies in package.json fail to install under node 4.1.1:

```
    "grunt-sass": "1.0.0"
    "phantom": "0.6.5"
```
See log below for details.

Downgrading the node version to 0.12.* avoids the problem. However, the README setup instructions say to run `brew install node` which as of recently will install node 4.1.1. So the long-term solution is probably to bump the package versions of grunt-sass and phantom to ones which are compatible with newer versions of node.

## Steps to reproduce
Run `npm install` with node version 4.1.1 installed.

## Environment
**Node version:** 4.1.1
**OS:** OS X Yosemite. Darwin Kernel Version 14.5.0: Wed Jul 29 02:26:53 PDT 2015; root:xnu-2782.40.9~1/RELEASE_X86_64

## Log output:
```
npm WARN engine karma@0.12.31: wanted: {"node":"~0.8 || ~0.10"} (current: {"node":"4.1.0","npm":"2.14.4"})
npm WARN peerDependencies The peer dependency karma@>=0.11.14 included from karma-coffee-preprocessor will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-copy will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-clean will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency karma@>=0.9 included from karma-spec-reporter will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.2 included from grunt-autoprefixer will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.2 included from grunt-jscs will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@>=0.4.0 included from grunt-contrib-uglify will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-watch will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency karma@>=0.9 included from karma-coverage will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.5 included from grunt-frequency-graph will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-webfontjson will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@0.4.x included from grunt-karma will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency karma@~0.12.0 included from grunt-karma will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.2 included from grunt-px-to-rem will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@>=0.4.5 included from grunt-scss-lint will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.1 included from grunt-bytesize will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-csdevmode will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN engine xmlbuilder@2.4.3: wanted: {"node":"0.8.x || 0.10.x || 0.11.x"} (current: {"node":"4.1.0","npm":"2.14.4"})

> cson@1.6.2 preinstall /Users/joseph_smith/dev/frontend-new/node_modules/load-grunt-config/node_modules/cson
> node ./cyclic.js


> ambi@2.2.0 preinstall /Users/joseph_smith/dev/frontend-new/node_modules/load-grunt-config/node_modules/cson/node_modules/ambi
> node ./cyclic.js


> typechecker@2.0.8 preinstall /Users/joseph_smith/dev/frontend-new/node_modules/load-grunt-config/node_modules/cson/node_modules/extract-opts/node_modules/typechecker
> node ./cyclic.js


> typechecker@2.0.8 preinstall /Users/joseph_smith/dev/frontend-new/node_modules/load-grunt-config/node_modules/cson/node_modules/ambi/node_modules/typechecker
> node ./cyclic.js


> js2coffee@0.3.5 preinstall /Users/joseph_smith/dev/frontend-new/node_modules/load-grunt-config/node_modules/cson/node_modules/js2coffee
> node ./cyclic.js


> weak@0.4.1 install /Users/joseph_smith/dev/frontend-new/node_modules/phantom/node_modules/dnode/node_modules/weak
> node-gyp rebuild

  CXX(target) Release/obj.target/weakref/src/weakref.o
In file included from ../src/weakref.cc:19:
../node_modules/nan/nan.h:261:25: error: redefinition of '_NanEnsureLocal'
NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
                        ^
../node_modules/nan/nan.h:256:25: note: previous definition is here
NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Handle<T> val) {
                        ^
../node_modules/nan/nan.h:661:13: error: no member named 'smalloc' in namespace 'node'
    , node::smalloc::FreeCallback callback
      ~~~~~~^
../node_modules/nan/nan.h:672:12: error: no matching function for call to 'New'
    return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
           ^~~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/node_buffer.h:31:40: note: candidate function not viable: no known conversion from 'uint32_t'
      (aka 'unsigned int') to 'enum encoding' for 3rd argument
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
/Users/joseph_smith/.node-gyp/4.1.0/include/node/node_buffer.h:43:40: note: candidate function not viable: 2nd argument ('const char *') would lose const
      qualifier
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
/Users/joseph_smith/.node-gyp/4.1.0/include/node/node_buffer.h:28:40: note: candidate function not viable: requires 2 arguments, but 3 were provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate, size_t length);
                                       ^
/Users/joseph_smith/.node-gyp/4.1.0/include/node/node_buffer.h:36:40: note: candidate function not viable: requires 5 arguments, but 3 were provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
In file included from ../src/weakref.cc:19:
../node_modules/nan/nan.h:676:12: error: no viable conversion from 'v8::MaybeLocal<v8::Object>' to 'v8::Local<v8::Object>'
    return node::Buffer::New(v8::Isolate::GetCurrent(), size);
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:210:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion
      from 'v8::MaybeLocal<v8::Object>' to 'const v8::Local<v8::Object> &' for 1st argument
class Local {
      ^
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:210:7: note: candidate constructor (the implicit move constructor) not viable: no known conversion
      from 'v8::MaybeLocal<v8::Object>' to 'v8::Local<v8::Object> &&' for 1st argument
class Local {
      ^
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:214:13: note: candidate template ignored: could not match 'Local' against 'MaybeLocal'
  V8_INLINE Local(Local<S> that)
            ^
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:326:13: note: candidate template ignored: could not match 'S *' against 'v8::MaybeLocal<v8::Object>'
  V8_INLINE Local(S* that)
            ^
In file included from ../src/weakref.cc:19:
../node_modules/nan/nan.h:683:26: error: no member named 'Use' in namespace 'node::Buffer'
    return node::Buffer::Use(v8::Isolate::GetCurrent(), data, size);
           ~~~~~~~~~~~~~~^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:221:5: error: assigning to 'v8::Primitive *volatile' from incompatible type 'v8::Value *'
    TYPE_CHECK(T, S);
    ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:414:12: note: in instantiation of function template specialization 'v8::Local<v8::Primitive>::Local<v8::Value>' requested here
    return NanEscapeScope(NanNew(v8::Undefined(v8::Isolate::GetCurrent())));
           ^
../node_modules/nan/nan.h:398:30: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEnsureLocal(val))
                             ^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:221:5: error: assigning to 'v8::Boolean *volatile' from incompatible type 'v8::Value *'
    TYPE_CHECK(T, S);
    ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:424:12: note: in instantiation of function template specialization 'v8::Local<v8::Boolean>::Local<v8::Value>' requested here
    return NanEscapeScope(NanNew(v8::True(v8::Isolate::GetCurrent())));
           ^
../node_modules/nan/nan.h:398:30: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEnsureLocal(val))
                             ^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:221:5: error: assigning to 'v8::Function *volatile' from incompatible type 'v8::Value *'
    TYPE_CHECK(T, S);
    ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:1514:12: note: in instantiation of function template specialization 'v8::Local<v8::Function>::Local<v8::Value>' requested here
    return NanEscapeScope(NanNew(handle)->Get(kCallbackIndex)
           ^
../node_modules/nan/nan.h:398:30: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEnsureLocal(val))
                             ^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:221:5: error: assigning to 'v8::Object *volatile' from incompatible type 'v8::Value *'
    TYPE_CHECK(T, S);
    ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:1632:12: note: in instantiation of function template specialization 'v8::Local<v8::Object>::Local<v8::Value>' requested here
    return NanEscapeScope(handle->Get(NanNew(key)).As<v8::Object>());
           ^
../node_modules/nan/nan.h:398:30: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEnsureLocal(val))
                             ^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:7367:3: error: assigning to 'v8::Integer *volatile' from incompatible type 'v8::Value *'
  TYPE_CHECK(T, S);
  ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../src/weakref.cc:93:3: note: in instantiation of function template specialization 'v8::ReturnValue<v8::Integer>::Set<v8::Value>' requested here
  NanReturnValue(NanNew<Integer>(None));
  ^
../node_modules/nan/nan.h:401:61: note: expanded from macro 'NanReturnValue'
# define NanReturnValue(value) return args.GetReturnValue().Set(_NanEnsureLocal(value))
                                                            ^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:7367:3: error: assigning to 'v8::Boolean *volatile' from incompatible type 'v8::Value *'
  TYPE_CHECK(T, S);
  ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../src/weakref.cc:99:3: note: in instantiation of function template specialization 'v8::ReturnValue<v8::Boolean>::Set<v8::Value>' requested here
  NanReturnValue(NanNew<Boolean>(!dead && obj->Delete(property)));
  ^
../node_modules/nan/nan.h:401:61: note: expanded from macro 'NanReturnValue'
# define NanReturnValue(value) return args.GetReturnValue().Set(_NanEnsureLocal(value))
                                                            ^
In file included from ../src/weakref.cc:18:
In file included from /Users/joseph_smith/.node-gyp/4.1.0/include/node/node.h:42:
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:7367:3: error: assigning to 'v8::Array *volatile' from incompatible type 'v8::Value *'
  TYPE_CHECK(T, S);
  ^~~~~~~~~~~~~~~~
/Users/joseph_smith/.node-gyp/4.1.0/include/node/v8.h:180:37: note: expanded from macro 'TYPE_CHECK'
    *(static_cast<T* volatile*>(0)) = static_cast<S*>(0);      \
                                    ^ ~~~~~~~~~~~~~~~~~~
../src/weakref.cc:135:3: note: in instantiation of function template specialization 'v8::ReturnValue<v8::Array>::Set<v8::Value>' requested here
  NanReturnValue(dead ? NanNew<Array>(0) : obj->GetPropertyNames());
  ^
../node_modules/nan/nan.h:401:61: note: expanded from macro 'NanReturnValue'
# define NanReturnValue(value) return args.GetReturnValue().Set(_NanEnsureLocal(value))
                                                            ^
12 errors generated.
make: *** [Release/obj.target/weakref/src/weakref.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "/usr/local/Cellar/node/4.1.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/joseph_smith/dev/frontend-new/node_modules/phantom/node_modules/dnode/node_modules/weak
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.1
gyp ERR! not ok

> phantomjs@1.9.18 install /Users/joseph_smith/dev/frontend-new/node_modules/phantomjs
> node install.js

PhantomJS detected, but wrong version 2.0.0 @ /usr/local/bin/phantomjs.
Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-macosx.zip
Saving to /var/folders/b2/4t1rxhdn7mg8gv_rgj5nl1w1c8l873/T/phantomjs/phantomjs-1.9.8-macosx.zip
Receiving...
| [=============---------------------------] 32% 0.0s|
Received 9187K total.
Extracting zip contents
Removing /Users/joseph_smith/dev/frontend-new/node_modules/phantomjs/lib/phantom
Copying extracted folder /var/folders/b2/4t1rxhdn7mg8gv_rgj5nl1w1c8l873/T/phantomjs/phantomjs-1.9.8-macosx.zip-extract-1443695212291/phantomjs-1.9.8-macosx -> /Users/joseph_smith/dev/frontend-new/node_modules/phantomjs/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /Users/joseph_smith/dev/frontend-new/node_modules/phantomjs/lib/phantom/bin/phantomjs
npm WARN optional dep failed, continuing weak@0.4.1

> fsevents@0.3.8 install /Users/joseph_smith/dev/frontend-new/node_modules/karma/node_modules/chokidar/node_modules/fsevents
> node-gyp rebuild

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node

> fsevents@0.3.8 install /Users/joseph_smith/dev/frontend-new/node_modules/babel/node_modules/chokidar/node_modules/fsevents
> node-gyp rebuild

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node

> ws@0.4.31 install /Users/joseph_smith/dev/frontend-new/node_modules/grunt-csdevmode/node_modules/socket.io/node_modules/engine.io/node_modules/ws
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o

> ws@0.4.31 install /Users/joseph_smith/dev/frontend-new/node_modules/grunt-csdevmode/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o

> ws@0.4.32 install /Users/joseph_smith/dev/frontend-new/node_modules/karma/node_modules/socket.io/node_modules/socket.io-client/node_modules/ws
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o

> node-sass@3.2.0 install /Users/joseph_smith/dev/frontend-new/node_modules/grunt-sass/node_modules/node-sass
> node scripts/install.js

Cannot download "https://github.com/sass/node-sass/releases/download/v3.2.0/darwin-x64-46_binding.node": HTTP error 404 Not Found

> node-sass@3.2.0 postinstall /Users/joseph_smith/dev/frontend-new/node_modules/grunt-sass/node_modules/node-sass
> node scripts/build.js

Building: /usr/local/Cellar/node/4.1.0/bin/node node_modules/pangyp/bin/node-gyp rebuild --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
gyp: /Users/joseph_smith/.node-gyp/4.1.0/common.gypi not found (cwd: /Users/joseph_smith/dev/frontend-new/node_modules/grunt-sass/node_modules/node-sass) while reading includes of binding.gyp while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/joseph_smith/dev/frontend-new/node_modules/grunt-sass/node_modules/node-sass/node_modules/pangyp/lib/configure.js:345:16)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "/usr/local/Cellar/node/4.1.0/bin/node" "/Users/joseph_smith/dev/frontend-new/node_modules/grunt-sass/node_modules/node-sass/node_modules/pangyp/bin/node-gyp" "rebuild" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /Users/joseph_smith/dev/frontend-new/node_modules/grunt-sass/node_modules/node-sass
gyp ERR! node -v v4.1.0
gyp ERR! pangyp -v v2.3.0
gyp ERR! not ok
Build failed
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/4.1.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.1.0
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE

npm ERR! node-sass@3.2.0 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@3.2.0 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/joseph_smith/dev/frontend-new/npm-debug.log
```

**Comment by joelochlann** [comment]
In fact you probably don't need to bump versions in `package.json` as the problem isn't actually with those package versions per se, but the versions of their dependencies which are locked down by `npm-shrinkwrap.json`. Specifically, `phantom` depends on `dnode` (locked to 1.2.1) and `grunt-sass` depends on `node-sass` which is locked to 3.2.0.

The `package.json` versions in phantom and grunt-sass allow some flexibility (`"node-sass": "^3.0.0"`, `"dnode": "~1.2.0"`) and in fact it works fine if we do a manual
```
npm install grunt-sass
npm install phantom
```
to ignore the shrinkwrap and install using the normal `package.json` conventions. (We end up with ` node-sass@3.3.3` and `dnode@1.2.2`).

So all that's needed is to save these updated dependencies in `npm-shrinkwrap.json`. The [npm shrinkwrap docs](https://docs.npmjs.com/cli/shrinkwrap) claim you can do it by running `npm install --save` for each package you want to update, but that doesn't seem to update `npm-shrinkwrap.json` for me. Not sure why.

**Comment by joelochlann** [comment]
Fixed by #13420
